packages/core/src/Kernel/LoadDiscoveryClasses.php

```diff
@@ -43,65 +44,70 @@ public function __invoke(): void
     public function build(): array
```

Comment:
This part was refactored so that we start from the locations instead of discovery classes.